.travis.yml

```
@@ -1,17 +1,28 @@
+dist: xenial
 language: python
 git:
   depth: 50 # Need old commits for determining version of untagged builds
-env: DJANGO=1.11.9
+env: DJANGO=1.11
 matrix:
   include:
   - python: 2.7
-  - python: 3.4
-  - python: 2.7 # Old Middleware classes
-    env: DJANGO=1.9
-  - python: 3.4 # Django 2 support
-    env: DJANGO==2.0
+  - python: 3.5
+  - python: 3.6
+  - python: 3.7
+  - python: 3.5 
+    env: DJANGO=2.1
+  - python: 3.6
+    env: DJANGO=2.1
+  - python: 3.7
+    env: DJANGO=2.1
+  - python: 3.5 
+    env: DJANGO=2.2
+  - python: 3.6
+    env: DJANGO=2.2
+  - python: 3.7
+    env: DJANGO=2.2
 install:
-  - pip install Django==$DJANGO
+  - pip install Django~=$DJANGO
```

Hi @asherf,

I'm sorry for I missed it initially, but you need to provide [a patch (aka micro) version](https://www.python.org/dev/peps/pep-0440/#semantic-versioning) in purpose [to pin a minor version](https://www.python.org/dev/peps/pep-0440/#compatible-release).
> For example, the following groups of version clauses are equivalent:
```
~= 2.2
>= 2.2, == 2.*

~= 1.4.5
>= 1.4.5, == 1.4.*
```


```suggestion
  - pip install Django~=${DJANGO}.0
```